torchgpe/bec2D/potentials.py

```diff
@@ -254,6 +257,7 @@ def get_alpha(self, psi: torch.tensor, time: float = None):
         self.eta = np.sqrt(self._lattice_depth(time))*self.eta_prefactor
 
         alpha = self.c1*self.eta*order/self.c6
+        alpha = (self.c1*self.eta*order + self._cavity_probe(time)*(self.Er/spconsts.hbar))/self.c6
```

Comment:
1) Shouldn't we remove line 259?
2) For real time evolution 260 will work, however for the imaginary time evolution we have "time"=None, which causes problems with callable cavity_probe(). I agree that it is not so useful in the imaginary time but we can allow in the imaginary time evolution to have stationary probes that do not change in time; constant cavity_probe(). That's why initially I had if clause discriminating with None value for the time and two different way of cavity_probe being calculated.